### PR TITLE
core,cmd/utils: bugfix for ropsten dump imports

### PR DESCRIPTION
**Pull request by bas-vk**
Importing a Ropsten exported chain currently fails because the official network genesis block is written instead of the Ropsten genesis block.

### CODE REVIEW

File: cmd/utils/flags.go [hunk 1]
```diff
@@ -925,11 +925,19 @@ func MakeChain(ctx *cli.Context, stack *node.Node) (chain *core.BlockChain, chai
 	chainDb = MakeChainDatabase(ctx, stack)
 
 	if ctx.GlobalBool(OlympicFlag.Name) {
+		_, err := core.WriteOlympicGenesisBlock(chainDb)
+		if err != nil {
+			glog.Fatalln(err)
+		}
+	}
+
+	if ctx.GlobalBool(TestNetFlag.Name) {
```
Comment (karalabe) [inline cmd/utils/flags.go]: Are we sure this is correct? Didn't this keep `--olympic` and drop `--testnet` (although `--olympic` would now load ropsten)?

Comment (karalabe) [inline cmd/utils/flags.go]: I'm a moron, nvm

Comment (obscuren) [inline cmd/utils/flags.go]: There are very few occasions where I agree with you, but this certainly is one of them :heart:

Comment (karalabe) [inline cmd/utils/flags.go]: Luv u

**Pull request merged**